oe_theme.theme

```diff
@@ -1522,3 +1523,88 @@ function oe_theme_preprocess_page(array &$variables): void {
     'id' => ['main-content'],
   ]);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_oe_organisation(&$variables): void {
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+
+  $entity = &$variables['entity'];
+  // Prepare fields.
+  $fields_values = ['name' => 'name', 'oe_acronym' => 'acronym'];
+  foreach ($fields_values as $field => $pattern_key) {
+    if ($entity->get($field)->isEmpty()) {
+      $variables[$pattern_key] = '';
```

Comment:
We don't need to set defaults to "pattern keys", Drupal theme system will do that for us. Also "pattern keys" are called "fields" in UI Patterns lingo, so I'd propose the following change here:

```php
  $fields_values = ['name' => 'name', 'oe_acronym' => 'acronym'];
  foreach ($fields_values as $field => $field) {
    if (!$entity->get($field)->isEmpty()) {
      $variables[$field] = $entity->get($field)->value;
    }
  }
```